docker/pulsar/Dockerfile

```diff
@@ -48,17 +48,13 @@ RUN sed -i "s|http://archive\.ubuntu\.com/ubuntu/|${UBUNTU_MIRROR:-mirror://mirr
      && apt-get update \
      && apt-get -y dist-upgrade \
      && apt-get -y install openjdk-11-jdk-headless netcat dnsutils less procps iputils-ping \
-                 python3 python3-dev python3-setuptools python3-yaml python3-kazoo \
-                 libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev \
+                 python3 python3-yaml python3-kazoo python3-pip \
```

Comment:
I think adding `python3-pip` will be pulling in GCC and a bunch of other stuff. That was the reason to fetch `get-pip.py` instead.

Reply:
I'll try to find a solution for that. Perhaps `--no-install-recommends` works.

Reply:
`apt-get install --no-install-recommends python3-pip` seems omit GCC and a lot of other stuff.

Could be verified with a "simulated" apt-get install by passing `-s`:
```
docker run --rm -it ubuntu:20.04 bash -c 'apt-get update && apt-get install -s --no-install-recommends python3-pip'
```
vs.
```
docker run --rm -it ubuntu:20.04 bash -c 'apt-get update && apt-get install -s python3-pip'
```